src/Resources/config/services.xml

```diff
@@ -25,24 +25,25 @@
             <argument type="service" id="htmldev.text_data_loader" />
         </service>
 
-        <service id="htmldev.abstract_data_loader" abstract="true">
-            <argument type="string">%htmldev.directory%</argument>
+        <service id="Zicht\Bundle\HtmldevBundle\Service\AbstractDataLoader" abstract="true">
+            <argument type="collection"/> <!-- set by config -->
```

Comment:
Ik dacht dat er in Symfony-land een soort aanduiding mogelijk was inmiddels waarmee je voor dit argument kunt aangeven dat deze dynamisch gezet wordt. Ik heb dit volgens mij een keer voorbij zien komen in een nieuwsbericht, maar ik kan het nergens meer terugvinden. Rings a bell?

Reply:
Ja ik ken het inderdaad. Is pas vanaf een bepaalde Sf versie (5.1 zelfs misschien pas?) en dit zou in Sf 3 projecten terecht kunnen komen, dus nog maar even niet gedaan nu :smile:

Reply:
Als je `<argument type="constant">null</argument>` gebruikt, dan laat phpstorm er achter zien wat de property name is die je hier zet.

Doorgaans zal de waarde toch aangepast worden vanuit de config oid, en dit maakt het wel lekker leesbaar.